requests/nodejs.yml

```
@@ -0,0 +1,8 @@
+action: cirun
+feedstocks:
+  - nodejs
+resources:
+  - cirun-openstack-cpu-xlarge
```

Just checking: do you need the extra RAM in xlarge? Maybe it's enough with a [smaller instance](https://github.com/Quansight/open-gpu-server?tab=readme-ov-file#cpu-runners)?

```suggestion
  - cirun-openstack-cpu-large
```

---

large is enough probably